docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -112,6 +112,37 @@ Specifying this setting in a transport profile in `elasticsearch.yml` will
 result in an error on startup.
 ====
 
+[discrete]
+[[saml-realm-nameid-changes]]
+.The `nameid_format` SAML realm setting has no default value.
```

Comment:
Should this be:
```suggestion
.The `nameid_format` SAML realm setting no longer has a default value.
```
?

Our breaking changes are usually described as a _change_ rather than a statement of the new behaviour.
That tends to be easier for people who are upgrading because it makes it easier to understand what is different and how that might affect them.

Reply:
SGTM